en/micro-integrator/docs/references/connectors/redis-connector/redis-connector-configuration.md

```diff
@@ -0,0 +1,36 @@
+# Setting up the Environment 
+
+The Redis connector allows you to access the Redis commands through the WSO2 EI. Redis stands for remote dictionary server. Redis store /serve that stores data as key-value pairs and this key-value store can be used as a database.
+
+## Setting up the environment
+
+Before you start configuring the Redis connector, you also need WSO2 MI, and we refer to that location as <PRODUCT_HOME>.
+
+To configure the SMPP connector, copy the following client libraries from the given locations to the `<PRODUCT_HOME>/lib` directory.
```

Comment:
Redis connector?

Reply:
there is nowhere to copy from. Instead should we say download from here (location) ?

Reply:
Updated correct download location.